tensorly/metrics/factors.py

```diff
@@ -28,16 +33,33 @@ def congruence_coefficient(matrix1, matrix2, absolute_value=True):
     congruence : float
     permutation : list
     """
-    num_cols = T.shape(matrix1)[1]
-    if T.shape(matrix1) != T.shape(matrix2):
-        raise ValueError("Matrices must have same shape")
-
-    matrix1 = matrix1 / T.norm(matrix1, axis=0)
-    matrix2 = matrix2 / T.norm(matrix2, axis=0)
-    all_congruences = T.dot(T.transpose(matrix1), matrix2)
-    if absolute_value:
-        all_congruences = T.abs(all_congruences)
-    all_congruences = T.to_numpy(all_congruences)
+    # Check if matrix1 and matrix2 are lists of the same length
+    if isinstance(matrix1, list):
```

Comment:
I think it is safer to do it the other way around: if tl.is_tensor(matrix1), make it into a list, otherwise, check that both have the same len (alternatively we could also check if it is iterable to not limit to list but accept list-like objects or tuples, but that would be slightly more involved/convoluted)